interface-definitions/system-option.xml.in

```diff
@@ -117,6 +117,26 @@
                <valueless/>
              </properties>
            </leafNode>
+           <leafNode name="root-partition-auto-resize">
+             <properties>
+               <help>Enable/disable root partition autoextension while system boot</help>
```

Comment:
Also, maybe add a hyphen to "auto-extension" for better readability?

Reply:
Sure